sql/catalyst/src/main/scala/org/apache/spark/sql/errors/QueryExecutionErrors.scala

```diff
@@ -247,6 +247,13 @@ object QueryExecutionErrors {
       messageParameters = Array(s"literal for '${v.toString}' of ${v.getClass.toString}."))
   }
 
+  def pivotColumnUnsupportedError(v: Any, dataType: DataType): RuntimeException = {
+    new SparkRuntimeException(
+      errorClass = "UNSUPPORTED_FEATURE",
+      messageParameters = Array(
+        s"pivoting by the value '${v.toString}' of the column data type '$dataType'."))
```

Comment:
SQL users can be not aware of Scala's types like `StructField`. Let's call `${dataType.catalogString}`
```suggestion
        s"pivoting by the value '${v.toString}' of the column data type '${dataType.catalogString}'."))
```

Reply:
@MaxGekk thanks. fixed.